students/ptfbanks/Lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,20 @@
+#Fixx, Buzz no Bang!
+print("This routine replaces numbers in a range (1 to n, you select n)")
+print ("which are divisible by values (v1, v2) you select.")
+print (" call this by entering fibu (v1, v2, n)")
+
+def fibu (a, b, n):
+	""" Replace each number divizable by variables a or b  respectivle with """
+	""" "Fizz" or "Buzz" or both, in the range of 1 through the variable n."""
+	for x in range(n+1):
+		if x%a == 0:
```

Comment:
your code is over-indented; pep8 recommends 4 spaces (no tabs).

you can configure your editor (like sublime) so that it uses 4 spaces for indentation, pycharm comes built-in with proper indent.